scripts/ide.sh

```diff
@@ -17,20 +17,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-TRUSTED_TOML="cmake/tomls/Cargo.sgx_trusted_lib.toml"
-UNTRUSTED_TOML="cmake/tomls/Cargo.sgx_untrusted_app.toml"
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
```

Comment:
Can we use ALL_CAPS for these variables?

Reply:
https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization @qinkunbao please read and see if capitalization is needed

Reply:
Do we have a shell style guides for reference? I found one: https://google.github.io/styleguide/shellguide.html. If it is worth to follow, we can put it into our docs.

Reply:
> https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization @qinkunbao please read and see if capitalization is needed

Thanks for the information. It looks like variables like `TRUSTED_TOML` and `UNTRUSTED_TOML` in this shell script should be the lower case as well, according to the convention.

Reply:
Some bash scripts under the `cmake/scripts` folder do not the following the convention.

For example,

https://github.com/apache/incubator-teaclave/blob/c09805e998174034e491e1b46258b7308dc0789d/cmake/scripts/sgx_link_sign.sh#L35-L38
https://github.com/apache/incubator-teaclave/blob/c09805e998174034e491e1b46258b7308dc0789d/cmake/scripts/gen_enclave_sig.sh#L30

Reply:
> Some bash scripts under the `cmake/scripts` folder do not the following the convention.
> 
> For example,
> 
> https://github.com/apache/incubator-teaclave/blob/c09805e998174034e491e1b46258b7308dc0789d/cmake/scripts/sgx_link_sign.sh#L35-L38
> 
> 
> https://github.com/apache/incubator-teaclave/blob/c09805e998174034e491e1b46258b7308dc0789d/cmake/scripts/gen_enclave_sig.sh#L30

Don't worry. We can improve this gradually. You can crate a new issue to track this legacy code style issue.